app/controllers/multifactor_auths_controller.rb

```diff
@@ -42,7 +42,7 @@ def update
     session[:level] = level_param
     @user = current_user
 
-    setup_mfa_authentication
+    @form_mfa_url = mfa_update_multifactor_auth_url(token: current_user.confirmation_token)
```

Comment:
When trying to update the mfa level with just webauthn, I get this error. I believe you need to remove the `mfa_update` from the require totp enabled before action.

<img width="865" alt="Screenshot 2023-06-15 at 11 31 55 PM" src="https://github.com/rubygems/rubygems.org/assets/42748004/a60d1456-605c-46b0-b628-52b916104c60">